etc/docker/helm/rucio-server/README.md

```diff
@@ -0,0 +1,73 @@
+# Rucio
+
+##  Data Management for science in the Big Data era.
+
+Rucio is a project that provides services and associated libraries for allowing scientific collaborations to manage large volumes of data spread across facilities at multiple institutions and organisations. Rucio has been developed by the ATLAS. experiment. It offers advanced features, is highly scalable and modular. Rucio is a data management solution that could cover the needs of different communities in the scientific domain (e.g., HEP, astronomy, biology).
+
+## QuickStart
+
+Download the tgz archive or clone this repository and run either of the two commands:
+
+```bash
+$ helm install rucio-server-0.1.0.tgz
+```
+
+```bash
+$ helm install rucio-server/
+```
+
+## Introduction
+
+This chart bootstraps a Rucio server deployment and service on a Kubernetes cluster using the Helm Package manager.
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
```

Comment:
what's the significance of the release name? is this tied to the release version mentioned above?

Reply:
A release is an actual installation of a chart on the kubernetes cluster. It just generally helps grouping together all of the resources created by this chart. It's possible to install multiple releases of the same chart on a cluster. It's not directly tied to the version number.